### ISSUE TITLE
Explanation missing

### ISSUE THREAD

**Issue by iblue** [matthewlloyd/Voron-Parametric#16]
I stumbled upon this project, because I wanted to change the fan height in the dragon toolhead cowling from 7 to 10 mm and noticed, that Voron is not really open source, it is just freeware. They are distributing the equivalent of binaries.

Anyways. It seems that this project has come to a halt, but the explanation that is announced in the readme is not yet posted. Could you maybe explain in short, what happened, because I would be quite interested to "reverse-engineer" the Voron?

**Comment by matthewlloyd** [comment]
You are absolutely correct in your conclusion: Voron is falsely advertised as "open source" when in reality it is, as you say, just freeware, supported by affiliate income, donations, partnerships with companies, revenue from the PIF program, and who knows what else.

Like many others, I was banned from the Voron (and Mainsail!) communities by the "Voron team", for, amongst other things I did in the interests of the open source community as a whole, suggesting it would be great if they released the CAD source with design history and not believing them when they claimed such files did not exist (they later admitted that for many parts, and for the entire V0, they did in fact exist, but they made a conscious decision not to share them.) Go figure! Perhaps one day I will find time to write up the full story, but it has not seemed worth the time, and you are only the second person to ask.

Voron is not an open source project: it is the continuation of a failed commercial enterprise (MZBot), and their team still thinks and acts like it was a company. Now, I am all for commercial enterprise - and I have nothing against trade secrets or private IP - but they should not be advertising themselves as "open source". Binary products derived from source files, are not source files.

Given all that, I decided to stop contributing my time for free in any way that directly or indirectly benefits the Voron team or community. I am now keeping my various hardware and software innovations to myself, and if any do ever get shared, they will be made available in the form of commercial products only.

If you are looking for a genuinely open source 3D printer, there are many other superior choices these days.

That said, if you want to reverse engineer some Voron STEP files and open pull requests, I would be happy to review them.

**Comment by exaiyo** [comment]
> Given all that, I decided to stop contributing my time for free in any way that directly or indirectly benefits the Voron team or community. I am now keeping my various hardware and software innovations to myself, and if any do ever get shared, they will be made available in the form of commercial products only.
> 
> If you are looking for a genuinely open source 3D printer, there are many other superior choices these days.
> 

Good to know. Do you have suggestions for other superior printers?

**Comment by matthewlloyd** [comment]
> Good to know. Do you have suggestions for other superior printers?

Sure. I'm not fully up to date but here are some suggestions. If I was buying anew, I'd go for the Bambu X1(C) if I just wanted to print and not have to build or tinker, the Prusa XL if I wanted something that works well out of the box while allowing for some tinkering, or the Annex K3 if I wanted to build something from scratch.

Fully open source (original source CAD files provided):

- Prusa believes in open source and provides Autodesk Inventor source files for all their printed parts. The XL or MK4 are both excellent choices. Input shaping for fast printing, load cell bed leveling, very nice UI, fantastic service and support.

Open STEP:

- RatRig V-Core 3.1. STEP provided. OnShape provided but in some cases may not be fully open source (original sketches and design history seemed lost last time I checked). Non-commercial license, so may not be good for anything beyond hobby use.
- HevOrt. F3D provided, but I don't think this includes original sketches and design history, it's just an assembly of STEP. Non-commercial license.
- VzBot. Open STEP. Non-commercial license.

Open STL - free but not open source:

- Annex K3. Extremely well engineered, very fast due to the cross gantry design, fully enclosed. Great team and community. While the software (Klipper) is of course open source like all the others above, and while STLs are provided for printed parts, there is no STEP or CAD source available. Non-commercial license, more restrictive than most.

Commercial:

- BambuLabs X1C or P1P. Can't be beaten for the out-of-box printing experience and technical innovations.
